perfkitbenchmarker/pkb.py

```diff
@@ -409,12 +416,14 @@ def RunBenchmarks(publish=True):
       static_virtual_machine.StaticVirtualMachine.ReadStaticVirtualMachineFile(
           fp)
 
+  successful_benchmarks = []
+  failed_benchmarks = []
```

Comment:
I think it would be helpful to preserve the order of execution and status (in case there was a temporal association).
What do you think about maintaining a list of `(benchmark, result)` pairs?